source/EvoPlanet/evoplanet.client/src/app/services/solar-system/solar-system.service.ts

```diff
@@ -12,14 +12,14 @@ export class SolarSystemService {
 
   constructor(private http: HttpClient) { }
 
-  //TODO: make RESTAPI call using this:
-  /*
+  //TODO: Change link if it doesn't work
+
 getSolarSystem(): Observable<PlanetSystem> {
-  return this.http.get<PlanetSystem>('localthost...');
+  return this.http.get<PlanetSystem>('https://localhost:7081/api/SolarSystem');
 }
-*/
 
 
+/*
```

Comment:
Avoid comments, especially /**/ comments (they are harder to review in version control).